src/main/java/com/xceptance/xlt/report/mergerules/UrlRequestFilter.java

```diff
@@ -45,7 +45,7 @@ public UrlRequestFilter(final String regex, final boolean exclude)
     {
         // we don't want to cache here due to the large variance in
         // urls, it is too costly to look things up with a lot of cache misses
-        super("u", regex, exclude, 0);
+        super("u", regex, exclude, 1000);
```

Comment:
The cache is in use now. Is this by accident? Anyway, we need to either adjust the comment or set the cache size back to 0.

Reply:
It is not longer expensive to look that up and we also cache non-hits. Seems to work. Larger cache does not show an effect.